openvdb/version.h

```diff
@@ -58,27 +58,19 @@
         #error expected OPENVDB_ABI_VERSION_NUMBER <= OPENVDB_LIBRARY_MAJOR VERSION_NUMBER
     #endif
 #else
-    // Older versions of the library used the macros OPENVDB_2_ABI_COMPATIBLE
-    // and OPENVDB_3_ABI_COMPATIBLE.  For now, continue to support them.
-    #if defined OPENVDB_2_ABI_COMPATIBLE ///< @todo deprecated
-        #define OPENVDB_ABI_VERSION_NUMBER 2
-    #elif defined OPENVDB_3_ABI_COMPATIBLE ///< @todo deprecated
-        #define OPENVDB_ABI_VERSION_NUMBER 3
-    #else
-        #define OPENVDB_ABI_VERSION_NUMBER OPENVDB_LIBRARY_MAJOR_VERSION_NUMBER
-    #endif
+    #define OPENVDB_ABI_VERSION_NUMBER OPENVDB_LIBRARY_MAJOR_VERSION_NUMBER
 #endif
 
 // If using an OPENVDB_ABI_VERSION_NUMBER that has been deprecated, issue a message
 // directive. Note that an error is also set in openvdb.cc which enforces stricter
 // behavior during compilation of the library. Both can be optionally suppressed
 // by defining OPENVDB_USE_DEPRECATED_ABI.
-#ifndef OPENVDB_USE_DEPRECATED_ABI
-    #if OPENVDB_ABI_VERSION_NUMBER == 4
-        PRAGMA(message("NOTE: ABI = 4 is deprecated, define OPENVDB_USE_DEPRECATED_ABI "
-            "to suppress this message"))
-    #endif
-#endif
+// #ifndef OPENVDB_USE_DEPRECATED_ABI
+//     #if OPENVDB_ABI_VERSION_NUMBER == 5
+//         PRAGMA(message("NOTE: ABI = 5 is deprecated, define OPENVDB_USE_DEPRECATED_ABI "
+//             "to suppress this message"))
+//     #endif
+// #endif
```

Comment:
Why comment this out?

Reply:
Because it’s a bit premature to deprecate ABI=5, we put this back when we release 8.0?

Reply:
Ah understood -  I would actually advocate marking ABI 5 as deprecated in the same set of changes as this, but looks like this might need some discussion? Lets discuss in the TSC tomorrow before merging this

Reply:
Sure